sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/functions/FunctionsTest.java

```diff
@@ -334,6 +334,43 @@ public void testDecimalRounding() {
                 ------
                  -0.12
                 (1 row)
+                
+                -- the following tests are from mysql
+                select cast('1.00000001335143196001808973960578441619873046875E-10' as decimal(30,15));
+                     decimal
+                -------------------
+                 0.000000000100000
+                (1 row)
+                
+                select ln(14000) as c1, cast(ln(14000) as decimal(5,3)) as c2, cast(ln(14000) as decimal(5,3)) as c3;
```

Comment:
I already forgot if we have ln on decimals.
If we don't, this is also checking casting from FP to decimal

Reply:
We don't have ln for decimal, ln casts to double and the output is a double. 

This test was taken from MySQL.

Reply:
maybe you should call it testRounding.